### DISCUSSION TITLE
离线全量导入，Write vertex label时进程被kill掉

### DISCUSSION THREAD

**Discussion by rkzhang95** [TuGraph-family/tugraph-db#79] [Q&A]
问题描述:  离线全量导入数据，在导入近10亿节点的图数据时，最后在 Write vertex label过程由于内存不足进程被kill掉
数据描述：测试数据，10亿节点（单类型person），40亿边（单边类型person-knows-person），文件大小102G
尝试方法：1: lgraph_import默认配置下，物理机内存64G和128G，进程会被kill掉
                  2: 在64G物理机下，限制bucket_memory_mb（512），限制n_sorter（1）以及n_packer（2），情况相同均会被kill
                  3: 创建图模式后online增量更新，速度太慢（包限制16M），预估用时83h

**Comment by knightast** [comment]
贴下完整的log?
